pandas/tests/indexes/datetimes/test_date_range.py

```diff
@@ -292,6 +292,15 @@ def test_construct_over_dst(self):
                             freq='H', tz='US/Pacific')
         tm.assert_index_equal(result, expected)
 
+    def test_construct_with_different_start_end_string_format(self):
+        # GH 12064
```

Comment:
not sure this is actually testing what is in the issue, as there were provided tz's there.

Reply:
tz aren't provided in these examples via the keyword arg. The issue was that `'2013-01-01 00:00:00+09:00'` vs `'2013/01/01 02:00:00+09:00'` (note the `-` vs the `/`) ended up produced different tzoffsets

https://github.com/pandas-dev/pandas/issues/12064#issue-127047394